protos/format.proto

```diff
@@ -91,6 +91,9 @@ message IndexMetadata {
 
   // Index name. Must be unique within one dataset version.
   string name = 3;
+
+  // The latest version of the dataset this index covers.
```

Comment:
`the version of the dataset this index was built on`?

Reply:
fixed